ui/analyse/package.json

```diff
@@ -20,17 +20,17 @@
     "@types/yaireo__tagify": "^4.0.1"
   },
   "dependencies": {
-    "@badrap/result": "^0.2.8",
-    "@yaireo/tagify": "^4.0.5",
+    "@badrap/result": "0.2.8",
+    "@yaireo/tagify": "~4.0.5",
     "ceval": "2.0.0",
     "chat": "2.0.0",
-    "chessground": "git+ssh://git@github.com:Mind-Sports-Games/chessground.git#v7.11.1-pstrat2.43",
+    "chessground": "file:/home/vincent/Desktop/code/chessground",
```

Comment:
I would change these all back to github in the commit, we try not to commit temp changes like this and leave the dependencies for local testing.